.github/workflows/express-build.yml

```diff
@@ -0,0 +1,110 @@
+# GitHub Action Workflow for building on Linux and Windows
+
+name: ExpressJS Express Build
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  
+  build:
+    name: Express Build
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ["0.10", "0.12", "1.8", "2.5", "3.3", "4.9", "5.12", "6.17", "7.10", "8.17", "9.11", "10.19", "11.15", "12.16"]
+        os: [ubuntu-latest, windows-latest]
+        experimental: [false]
+        node-mirror: ["https://nodejs.org/dist/"]
+        include:
+          - node-version: 13
+            os: ubuntu-latest
+            node-mirror: "https://nodejs.org/download/nightly"
+            experimental: true
+          - node-version: 13
+            os: windows-latest
+            node-mirror: "https://nodejs.org/download/nightly"
+            experimental: true
+            
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Set up node using nvm
```

Comment:
Why are you opting for this as opposed to the `setup-node` action provided by the Actions team?

Reply:
Because the setup-node action provided by the actions team has the same problem that the Node Tool installer task on Azure Pipelines has:  It doesn't support Node 0.10 - 3.3.  If you try and use the setup-node action on those versions, it fails.  This action used nvm to install the appropriate node version and works over all node versions that I tested.

Reply:
Good news is we are dropping all those versions in 5.0. So if that is the only reason, perhaps we target this pr to 5.0 and forget about those versions.

Reply:
Oh!  Yeah I did run into that before, we need to get them to fix that.  

@bdougieYO @adrianmg @chrisrpatterson Who do we need to contact to get these node versions supported by the setup action?

Reply:
Edit: @chrispat Sorry I tagged the wrong handle.

Reply:
While I agree it is good we will be dropping them, we still need the tests to run for atleast a year.  And if they cannot add support it would block us from moving for a year.  Not optimal IMO.

Reply:
Yea, I was thinking about that after my comment. It would be nice to have this on the 4.x line.

Reply:
> @bdougieYO @adrianmg @chrisrpatterson Who do we need to contact to get these node versions supported by the setup action?

👋🏾 @chrispat is the correct person to get eyes on it. 

In regards to getting nvm working with setup-node, Let me know if this thread is helpful: https://github.com/actions/setup-node/issues/32.

Reply:
Never sure who to tag, just tagged all the GH folks I have chatted about actions with.  I will direct these questions at @chrispat in the future, thanks!

Reply:
I'll also reach out internally to @chrispat.  [When I brought this up to previously with the Azure Pipelines task](https://github.com/microsoft/azure-pipelines-tasks/issues/8987), it got closed unfixed, saying they weren't going to support iojs at this time.

Reply:
regardless, I suspect if they do fix it, it might be a while out. We could use the current install tool I am using until they fix it, then switch. That would allow you to move forward and target older lines.

Reply:
Yeah, if they continue to say no to supporting what we support on Express, I am good with using this alternate approach.  That said, we probably want to chat with @dcodeIO about their support here and make sure they will be amenable to our needs.

Reply:
Also to be clear, I would strongly support the actions team adopting this methodology so that the also support the official aliases we are working on.  @dcodeio is aware of this effort from our conversation here: https://github.com/pkgjs/nv/pull/5

Reply:
Interesting. Initially created the action because I needed aliases for rolling testing and mirrors. Wasn't aware of the Node 0.10 - 3.3 problem until now. Perhaps, if you guys depend on it as well for the time being, I can add collaborators? Should probably add more tests then so we don't break each other :)

Reply:
I was going to try and roll my own action to do this before I found @dcodeIO's action. I'm going to push to get the Actions team to update the Node tool, but I advocate for working to keep @dcodeIO  action stable for all of us, as long as it doesn't mess up any plans they might have.